infrastructure/webdriver/bidi/subscription.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Test console log are present</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script>
+    promise_test(async () => {
+        const some_message = "SOME MESSAGE";
+        // Subscribe to `log.entryAdded` BiDi events. This will not add a listener to the page.
+        await test_driver.bidi.log.entry_added.subscribe();
+        // Add a listener for the log.entryAdded event. This will not subscribe to the event, so the subscription is
+        // required before. The cleanup is done automatically after the test is finished.
+        const log_entry_promise = new Promise(resolve => test_driver.bidi.log.entry_added.on(resolve));
```

Comment:
Do you think that turning the `on()` callback into a promise is going to be the most common pattern, or at least a very common pattern? If so, what do you think about adding a `once()` convenience method returning a promise, similar to https://nodejs.org/api/events.html#eventsonceemitter-name-options?

Reply:
good idea. Added.

Reply:
done